precheck.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+sbt clean coverage test it:test scalafmtCheckAll coverageReport
```

Comment:
This is what I had to change for the the new version of sbt.
Works the same as before and scans production code and tests.